lib/cocoapods-core/specification.rb

```diff
@@ -223,6 +233,18 @@ def subspec?
 
     # @!group Dependencies & Subspecs
 
+    # @return [Bool] If this specification is a libary (app or test) specification.
```

Comment:
similar comment here, I think the terminology is a bit confusing but maybe its just me

Reply:
I'll update these comments to make them a little more clear, thanks!